integr8ly/cloud-resource-operator
# Update seed targets

## Overview

During development we often have to switch between provisioning managed and workshop types. To help speed up this process this change allows you to seed the `type` you want

## Verification

- Clone this branch
- Run `make cluster/prepare`
- Run make seed targets
- Ensure CR's are created as expected
